### ISSUE TITLE
ui: Authentication radio buttons are not reachable by keyboard

### ISSUE THREAD

**Issue by jgwhite** [hashicorp/waypoint#2147]
**Describe the bug**
On the project repository settings pages, the authentication radio buttons are not reachable by keyboard.

**Steps to Reproduce**
1. Visit the project repository settings page (i.e. `/default/my-project/settings`)
2. Try tabbing between fields
3. Observe you cannot reach the authentication radio buttons

https://user-images.githubusercontent.com/34030/130968385-05d8052d-7723-474e-8b6c-0f6718f39109.mp4

**Expected behavior**
You can tab to and between the authentication radio buttons, following all the normal browser behaviors for radio buttons.

**Waypoint Platform Versions**
* Waypoint Version: 0.5.1
